services/IoT/applications/libraries/python.md

```diff
@@ -674,7 +669,7 @@ The following code sample provides an outline of the format required for this me
 
 ### Retrieving all devices
 
-Use the *getAllDevices()* method to retrieve all devices within an organization on the {{site.data.keyword.iot_short_notm}}.
+Use the *getAllDevices()* method to retrieve all devices that are in an organization on the {{site.data.keyword.iot_short_notm}}.
 
 ```
 	  apiCli.getAllDevices({'typeId' : deviceTypeId})
```

Comment:
Fixed.

"To update a device property, define  the 'deviceInfo` parameter before you call the method."